### DISCUSSION TITLE
Can't dismiss all notifications from Android notifications menu at the same time

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#63199] [Mobile]
GitHub for Android version 1.125.0

I have two Android phones with the same version of GitHub installed. I swap between them. But that means any notifications I see on one I've probably seen on the other. 

If it's just one notification, it isn't a big deal, but when there are a whole bunch, I'd like to be able to swipe right to dismiss them all at once. I can for a number of other Android apps, but for some reason the GitHub app notifications don't seem to group/be dismissible as a group.

**Comment by aconsuegra** [comment]
Thanks for your feedback @jsoref. We are planning to add synchronization of notifications between devices, but we don't have an estimation about when it will be ready.

Regarding grouping, we are grouping notifications that belong to the same _thread_ (i.e. several events in the same issue, or several reviews in the same pull request). How would you group them instead?

**Reply by jsoref** [reply]
Afaict, Facebook and Facebook Lite appear to group all things together, so I can dismiss all Facebook/Facebook Lite events at once.

Google Photos appears to group by event type, so I think each `Similar Shots` is grouped together but separate from `Move ... photos to archive?`.

Gmail groups all email items by account together, so I can dismiss all mail from one account while retaining notifications for a different account.

I think I'd want to group by type. So, all PR notifications together, all Issue (* a PR is not an issue, I don't care about the underlying APIs, and I know very well that github devs confuse this as I'm in the middle of writing a PR to fix at least one github action that can't consistently get this right) notifications together, all Discussion notifications together, all Sudo Mode requests together.

**Reply by jsoref** [reply]
Re synchronizing dismissing notifications, I'm not sure I'd actually want that. If someone uses a different device to sudo mode auth, I'd actually want to know especially if I didn't realize I left a device unlocked and in a position for someone to be able to do that.

For normal notifications, yes, it'd be nice to have notifications go away if i've marked them as read anywhere (web ui, reading them in an email, ...)
